modules/side-panel/react/stories/stories.tsx

```diff
@@ -146,7 +146,7 @@ class SidePanelWrapper extends React.Component<{}, SidePanelState> {
   };
 
   private handleBreakpoint = (open: boolean) => {
-    console.warn('called');
+    console.warn('handle breakpoint called');
```

Comment:
I just added a more description console.

Reply:
I've seen some stories use `@storybook/addon-actions`. Do we have consensus on this? I don't know what this means for Component Story Format: https://medium.com/storybookjs/component-story-format-66f4c32366df

I think it is fine to leave this as a `console.warn` and leave the discussion for a bigger decision where we change stories to be consistent.